config/initializers/flipper.rb

```diff
@@ -20,12 +21,18 @@
   end
 end
 
-# Labeling what flipper calls "actors" as "users" in the UI
+# Modify Flipper::UI::Configuration to accept a custom view path.
+Flipper::UI::Configuration.prepend(FlipperExtensions::ConfigurationPatch)
+
 Flipper::UI.configure do |config|
+  config.custom_views_path = Rails.root.join('lib', 'flipper', 'views')
```

Comment:
Created a `views` directory within `lib/flipper` to store all our custom Flipper templates.